Tally attempted corrections (#58)
bcgsc/tigmint: README.md

+ `mapq=0`: Mapping quality threshold
+ `trim=0`: Number of bases to trim off contigs following cuts
+ `t=8`: Number of threads

> + `ac=3000`: Minimum contig length (bp) for tallying attempted corrections. This is for assessing the accuracy of Tigmint, and will not affect its performance.

Reviewer: ```suggestion
+ `ac=3000`: Minimum contig length (bp) for tallying attempted corrections. This is for logging purposes only, and will not affect the performance.
```
Just to avoid mentioning 'accuracy' in the README
